### PR TITLE
Task0

### PR DESCRIPTION
**Pull request by AlexandrShlyaeu**
Preparation tasks fot "AUTOMATED TESTING MENTORING PROGRAM"

### CODE REVIEW

File: Task0/task0.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+function summ(arr) {
+  const result = arr.reduce((sum, current) => sum + current, 0);
+  return result;
+}
+
+function addWithDelay(params) {
+  return new Promise((resolve, reject) => {
+    if (params.every(element => element >= 0)) {
+      resolve(
```
Comment (vitalics) [inline Task0/task0.js]: `resolve` must be inside `setTimeout` function

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  "dependencies": {
+    "babel-eslint": "^10.0.1",
+    "eslint-config-airbnb-base": "^13.1.0"
+  },
+  "devDependencies": {
+    "eslint": "^5.8.0",
+    "eslint-config-prettier": "^3.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-prettier": "^3.0.0",
+    "prettier": "^1.14.3"
```
Comment (vitalics) [inline package.json]: `prettier` + `eslint` = 💩 . Use only `eslint`

File: Task0/task0.js (hunk 1 above)
Comment (AlexandrShlyaeu) [reply Task0/task0.js]: done

File: package.json (hunk 2 above)
Comment (AlexandrShlyaeu) [reply package.json]: done, but we are using "prettier + eslint" in our project and it's looks good enough.

File: Task0/task0.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+function summ(arr) {
+  const result = arr.reduce((sum, current) => sum + current, 0);
+  return result;
+}
+
+function addWithDelay(params) {
+  return new Promise((resolve, reject) => {
+    if (params.every(element => element >= 0)) {
+      setTimeout(() => {
```
Comment (vitalics) [inline Task0/task0.js]: i think you need use `setTimeout` for wrap full promise
``` js
return new Promise((resolve, reject) => {
setTimeout(()=>{
// full code promise here
}, 1000)
}
```

**Pull request merged**